src/decision/test/lidar-decision-test.cpp

```diff
@@ -6,10 +6,65 @@
 
 #include <LidarDecision.h>
 #include <gtest/gtest.h>
+#include <vector>
+
+TEST(LidarDecision, obstacle_in_range){
+    sensor_msgs::LaserScan test_scan;
+    test_scan.angle_max = 3.1415;
+    test_scan.angle_min = 0.0000;
+    test_scan.angle_increment = 0.01;
+    int vector_size = 300;
```

Comment:
Vectors have a function called size (ie. `vec.Size()`) which will return the size of the vector. Please use this instead, it makes it a lot easier to change and read the code.